polkadot/node/core/backing/src/lib.rs

```diff
@@ -2218,3 +2226,17 @@ fn handle_get_backable_candidates_message(
 	tx.send(backed).map_err(|data| Error::Send(data))?;
 	Ok(())
 }
+
+fn validation_request_ttl(mode: ProspectiveParachainsMode) -> Option<Instant> {
+	const MILLISECS_PER_BLOCK: u64 = 6000;
```

Comment:
Don't we have this constant usefully exposed somewhere already? I guess not, because it is up to the runtime to define this, but this also means that this is not actually sound. Given that this code here should be generic over any runtime. A better approach would be to give the ttl directly in form of block numbers, instead of going via a proxy which is time.